src/aspects/cfn-include-reporter.ts

```diff
@@ -0,0 +1,18 @@
+import type { IAspect } from "aws-cdk-lib";
+import { Annotations } from "aws-cdk-lib";
+import { CfnInclude } from "aws-cdk-lib/cloudformation-include";
+import type { IConstruct } from "constructs";
+import { LibraryInfo } from "../constants";
+
+/**
+ * An Aspect to remind users to check for stateful resources when migrating a YAML/JSON template.
+ */
+export class CfnIncludeReporter implements IAspect {
+  public visit(node: IConstruct) {
+    if (node instanceof CfnInclude) {
```

Comment:
Not blocking but this makes me think, are there any edge cases where `CfnInclude` is used that are not part of a migration?

Reply:
I'm not familiar with any instance of this. I'd suggest it's bad practice to do so too, as it's mixing two IaC methods.